INSTRUCTION.md

```diff
@@ -0,0 +1,10 @@
+1. Instruction on  how to deploy 
```

Comment:
The title 'Instruction on how to deploy' should be formatted consistently. Consider using a consistent heading style, such as '## Instruction on How to Deploy'.